hw2/fakeDb.js

```diff
@@ -0,0 +1,18 @@
+const DB = [

```

Comment:
Don't name it just "DB", be more specific what's inside, this looks to me as collection/table/database of Users

Reply:
This is not FakeDB, this is a real In-memory DB;